vulcan/_api.py

```diff
@@ -123,30 +131,34 @@ async def _request(
 
                 # check for the presence of a b64 string preceded with ': '
                 if status["Code"] == 100 and ": " in status["Message"]:
-                    raise VulcanAPIException("Invalid signature values.")
+                    raise InvalidSignatureValuesException
```

Comment:
I think you should construct these classes, not just raise them (`raise InvalidSignatureValuesException()`).